python/Ganga/__init__.py

```diff
@@ -299,27 +299,13 @@ def getLCGRootPath():
 lcg_config = makeConfig('LCG', 'LCG/gLite/EGEE configuration parameters')
 #gproxy_config = getConfig('GridProxy_Properties')
 
-# set default values for the configuration parameters
-lcg_config.addOption(
-    'EDG_ENABLE', False, 'DEPRECATED enables/disables the support of the EDG middleware')
-
-lcg_config.addOption('EDG_SETUP', '/afs/cern.ch/sw/ganga/install/config/grid_env_auto.sh',
-                 'DEPRECATED sets the LCG-UI environment setup script for the EDG middleware',
-                 filter=Ganga.Utility.Config.expandvars)
-
-lcg_config.addOption(
-    'GLITE_ENABLE', True, 'Enables/disables the support of the GLITE middleware')
```

Comment:
Yes but I would imagine it's easier when we first introduce this to give some error about the option being deprecated and construct it to be a read-only object so that if a user tires to set it an exception is thrown?
That way Ganga won't start with an old config which will probably lead to confusion about what parameter has been set and what hasn't.
I imagine the users should read the errors and should read the release notes but I'm of the opinion that if we want to enforce this we need to ensure that Ganga won't start with a bad configuration setup.